source/code/plugins/security_baseline_lib.rb

```diff
@@ -0,0 +1,116 @@
+require 'json'
+require 'securerandom' # SecureRandom.uuid 
+
+require_relative 'oms_common'
+
+module OMS
+    class SecurityBaseline
+
+        @@log = nil
+        def self.log= (value)
+	        @@log = value
+        end
+
+        # ------------------------------------------------------
+        def self.transform_and_wrap(results, host, time)
+            if results == nil
+                @@log.error "Security Baseline Assessment failed; Empty input"
+                return nil, nil
+            end
+
+            if results["results"] == nil 
+                @@log.error "Security Baseline Assessment failed; Invalid input:" + results.inspect
+                return nil, nil
+            end
+
+            results["assessment_id"] = SecureRandom.uuid      
+    
+            asm_baseline_results = results["results"]
+            scan_time = results["scan_time"]
+            assessment_id = results["assessment_id"]
```

Comment:
In general assessment_id value should be added to result file and this is only temporary solution.
Also results["assessment_id"] value used in line 54 inside calculate_summary method.